Cargo.toml

```diff
@@ -3,18 +3,22 @@ authors = ["Marc Brinkmann <git@marcbrinkmann.de>"]
 description = "Linux SocketCAN library. Send and receive CAN frames via CAN bus on Linux."
 documentation = "https://docs.rs/socketcan"
 license = "MIT"
+edition = "2018"
 name = "socketcan"
 repository = "https://github.com/mbr/socketcan-rs"
 version = "2.0.0"
 
+[features]
+default = ["netlink"]
+netlink = ["neli"]
+vcan_tests = ["netlink"]
+
 [dependencies]
 byte_conv = "0.1.1"
-hex = "^0.2"
-itertools = "^0.4"
-libc = "^0.2"
-netlink-rs = { git = "https://github.com/mbr/netlink-rs", rev = "01cba6fcc7b11917890bc3d2b4635009fde8082c" }
-nix = "^0.5"
-try_from = "0.2.0"
-
-[features]
-vcan_tests = []
+hex = "0.4"
+itertools = "0.10"
+libc = "0.2"
+nix = "0.23"
+clap = "2.33"
```

Comment:
I'd consider moving the `bin/can.rs` to `examples/can.rs` or creating a workspace with two separate crates to avoid the dependencies on `anyhow` and `clap` for projects that use the socketcan crate only for the library part.

Reply:
Yes, you're probably right. Normally I don't worry about it, but for a small, embedded crate like this the overhead is significant. I prefer to keep examples short and easy to understand, and keep what I consider "useful utilities" separate. A workspace seems a bit too complicated. Maybe just hide the utility(s) behind a feature flag?

Also I was debating the name. "can" seens a bit generic, though I want to keep it short and simple. "rcan"? "canrs"?

Reply:
I went with a feature flag.